### PR TITLE
Added support for @QueryParam annotation

### PR DESCRIPTION
**Pull request by SergeKorobko**
Hi, 

I've made some little changes in your plugin - now it knows how to deal with jax-rs @QueryParam annotation.

It would be great to have this feature in master. 

Thanks.

### CODE REVIEW

**Comment by misternerd** [comment]
@SergeKorobko Sorry for the delay, I was on vacation. I'll try to review your PR over the weekend.

File: src/main/java/com/misternerd/resteasytox/RestServiceLayout.java [hunk 1]
```diff
@@ -213,6 +214,12 @@ else if(jaxWsAnnotations.context.equals(annotation.annotationType()))
 					logger.debug("Context parameter will be discarded for method=" + method.getName());
 					continue parameterLoop;
 				}
+				else if(jaxWsAnnotations.queryParam.equals(annotation.annotationType()))
+				{
+					queryParams.add(new MethodParameter(getValueForJaxWsAnnotation(annotation), parameterTypes[i]));
+					logger.debug("Query parameter will be discarded for method=" + method.getName());
```
Comment (misternerd) [inline src/main/java/com/misternerd/resteasytox/RestServiceLayout.java]: @SergeKorobko Please remove the logger here, looks like it is left over

File: src/main/java/com/misternerd/resteasytox/javascript/ResteasyToJavascriptConverter.java [hunk 2]
```diff
@@ -331,8 +352,14 @@ private void createHttpMethodCall(ServiceMethod serviceMethod, JavascriptPublicM
 		// GET requires no body
 		if(GET == serviceMethod.httpMethod)
 		{
-			method.addBody("var request = restClient.getRequest(PATH + '%s', headerParams, pathParams, '%s', '%s', %s);",
-				serviceMethod.path, serviceMethod.requestContentType, serviceMethod.responseContentType, returnType);
+			String urlStr;
+			if(!serviceMethod.queryParams.isEmpty()) {
```
Comment (misternerd) [inline src/main/java/com/misternerd/resteasytox/javascript/ResteasyToJavascriptConverter.java]: @SergeKorobko I think it makes sense to support query params for all types of request. Please move this URL generating block up and use it for all method types.

**Review by misternerd — CHANGES_REQUESTED** [review]
Hey Serge, I added a few comments to your PR. It would be great if you can fix them or write me feedback on them. Thanks!

File: src/main/java/com/misternerd/resteasytox/javascript/ResteasyToJavascriptConverter.java (hunk 2 above)
Comment (SergeKorobko) [reply src/main/java/com/misternerd/resteasytox/javascript/ResteasyToJavascriptConverter.java]: @misternerd : done.

**Review by SergeKorobko — COMMENTED** [review]
Hi, @misternerd. Did you had a chance to take a look on merge request?

**Comment by misternerd** [comment]
Hi @SergeKorobko, finally got around to do this, thanks for your patience!

**Comment by SergeKorobko** [comment]
Hi @misternerd, thank you!

**Pull request merged**